addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyScanRule.java

```diff
@@ -750,6 +769,13 @@ public List<Alert> getExampleAlerts() {
                                         + "fonts.gstatic.com; frame-ancestors 'none'; worker-src 'self'; form-action 'none'")
                         .build());
         alerts.add(buildBothAlert().build());
+        alerts.add(
+                buildNofallbackAlert(
+                                HttpFieldsNames.CONTENT_SECURITY_POLICY,
+                                "connect-src *; default-src 'self'; form-action 'none'",
+                                Constant.messages.getString(
+                                        MESSAGE_PREFIX + "nofallback.otherinfo", FRAME_ANCESTORS))
```

Comment:
This should be in `buildNofallbackAlert` instead of duplicating and potentially getting out of sync.

Reply:
It would be preferable to use/keep the `MESSAGE_PREFIX`.